internal/cloudapi/v2/openapi.v2.yml

```diff
@@ -507,11 +507,14 @@ components:
       type: string
       enum:
         - aws
-        - gcp
         - azure
         - edge-commit
         - edge-container
         - edge-installer
+        - gcp
+        - image-installer
+        - qcow2
+        - vmdk
```

Comment:
This is for vSphere? Call in `vsphere`?

Reply:
agreed, should refer to the targetplatform rather than the on disk format of the image